File: Chapter 02/C Programs/2_17.c
#include<stdio.h>
void main()
{
int quantity, amt;
float x;
printf("Enter no of products you purchased: ");
scanf("%d", &quantity);
amt=quantity * 120;
if(amt>5000)
{
x= amt-0.5*amt;
printf("Pls pay %f", x);
}
else if (amt>  1000)
{
x= amt-0.2*amt;
printf("Pls pay %f", x);
}
else
{
printf("Pls pay %d", amt);
}
}

File: Chapter 02/Python programs/2_17.py
quantity=int(input("Enter no of products you purchased "))
amt=quantity * 120
if amt>5000:
    x= amt-0.5*amt
    print("Pls pay ", x)
elif amt>  1000:
    x= amt-0.2*amt
    print("Pls pay ", x)
else:
    print("Pls pay ", amt)
